### PR TITLE
typos noticed from spell-check

### PR DESCRIPTION
**Pull request by vjcharles**
I'm pretty sure these are safe to change but want to verify.

### CODE REVIEW

File: schema.yml [hunk 1]
```diff
@@ -647,7 +647,7 @@ szip:
     - manually_mapped
 urbanization:
   type: text
-  human: Parcel Urbanizacion
+  human: Parcel Urbanization
```
Comment (hampelm) [inline schema.yml:650]: @Blake-Loveland Thoughts on US vs ES spelling of Urbanizacion?

**Review by hampelm — APPROVED** [review]
Looks good to me, question for @Blake-Loveland. Thanks Vincent!

File: schema.yml (hunk 1 above)
Comment (Blake-Loveland) [reply schema.yml:650]: I would vote for ES.

**Review by Blake-Loveland — APPROVED** [review]
I did vote to make the urbanization the ES version, but I am ok either way. Thank you for catching my typos, I will make sure to try and catch them going forward.

**Pull request merged**